packages/dart_frog_cli/test/src/daemon/logger_test.dart

```diff
@@ -0,0 +1,307 @@
+        idGenerator: idGenerator,
+      ).alert('alert');
+
+      expect(daemonEvents.length, 1);
+
+      expect(
+        daemonEvents.last,
+        const DaemonEvent(
+          domain: 'test',
+          event: 'loggerAlert',
+          params: {
+            'meta-information1': true,
+            'message': 'alert',
+          },
+        ),
+      );
+    });
+
+    test('chooseAny', () {
+      final logger = DaemonLogger(
+        domain: 'test',
+        params: {},
+        sendEvent: sendEvent,
+        idGenerator: idGenerator,
+      );
+
+      expect(
+        () => logger.chooseAny('choose any of this', choices: ['oi', 'ai']),
+        throwsA(unsupportedErrorWithMessage('chooseAny')),
+      );
+    });
+
+    test('chooseOne', () {
+      final logger = DaemonLogger(
+        domain: 'test',
+        params: {},
+        sendEvent: sendEvent,
+        idGenerator: idGenerator,
+      );
+
+      expect(
+        () => logger.chooseOne('choose one of this', choices: ['oi', 'ai']),
+        throwsA(unsupportedErrorWithMessage('chooseOne')),
+      );
+    });
+
+    test('confirm', () {
+      final logger = DaemonLogger(
+        domain: 'test',
+        params: {},
+        sendEvent: sendEvent,
+        idGenerator: idGenerator,
+      );
+
+      expect(
+        () => logger.confirm('confirm this'),
+        throwsA(unsupportedErrorWithMessage('confirm')),
+      );
+    });
+
+    test('delayed/flush', () {
+      DaemonLogger(
+        domain: 'test',
+        params: params,
+        sendEvent: sendEvent,
+        idGenerator: idGenerator,
+      )
+        ..delayed('delayed1')
+        ..delayed('delayed2')
+        ..flush();
+
+      expect(daemonEvents.length, 2);
+
+      expect(
+        daemonEvents.first,
+        const DaemonEvent(
+          domain: 'test',
+          event: 'loggerInfo',
+          params: {
+            'meta-information1': true,
+            'message': 'delayed1',
+          },
+        ),
+      );
+
+      expect(
+        daemonEvents.last,
+        const DaemonEvent(
+          domain: 'test',
+          event: 'loggerInfo',
+          params: {
+            'meta-information1': true,
+            'message': 'delayed2',
+          },
+        ),
+      );
+    });
+
+    test('detail', () {
+      DaemonLogger(
+        domain: 'test',
+        params: params,
+        sendEvent: sendEvent,
+        idGenerator: idGenerator,
+      ).detail('detail');
+
+      expect(daemonEvents.length, 1);
+
+      expect(
+        daemonEvents.last,
+        const DaemonEvent(
+          domain: 'test',
+          event: 'loggerDetail',
+          params: {
+            'meta-information1': true,
+            'message': 'detail',
+          },
+        ),
+      );
+    });
+
+    test('err', () {
+      DaemonLogger(
+        domain: 'test',
+        params: params,
+        sendEvent: sendEvent,
+        idGenerator: idGenerator,
+      ).err('err');
+
+      expect(daemonEvents.length, 1);
```

Comment:
```suggestion
      expect(daemonEvents.length, equals(1));
```
And everywhere else this could apply :)